core/app/models/workarea/navigation/breadcrumbs.rb

```diff
@@ -13,7 +13,7 @@ def self.from_global_id(via, last: nil)
 
       def initialize(navigable, last: nil)
         @navigable = navigable
-        @last = Taxon.new(name: last) if last.present?
+        @last = Taxon.new(name: last.name, navigable: last) if last.present?
```

Comment:
Can we have this gracefully degrade so keep the same API? Then we don't need to change the `ProductViewModel`.
```suggestion
        @last = if last.try(:navigable?)
          Taxon.new(name: last.name, navigable: last)
        elsif last.present?
          Taxon.new(name: last)
        end
```

Reply:
I guess I'm OK with this but again this defeats the purpose of the fix to schema.org helpers, which is what this ticket was about in the first place. Definitely would help with backwards-compatibility in a patch, though, so I think this is OK...but I can't undo the changes to ProductViewModel as a result since it needs to pass in the full model so the final Taxon knows what URL to generate in `storefront_path_for`.